src/server/auth/server/testing/auth_integration_test.go

```diff
@@ -274,6 +268,18 @@ func TestDebug(t *testing.T) {
 	})
 }
 
+func TestDebug(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	t.Parallel()
+	c, _ := minikubetestenv.AcquireCluster(t, defaultTestOptions)
+	tu.ActivateAuthClient(t, c)
+	for _, projectName := range []string{pfs.DefaultProjectName, tu.UniqueString("project")} {
+		testDebug(t, c, projectName, tu.UniqueString("repo"))
```

Comment:
I would prefer `t.Run(projectName, func(t *testing.T) { testDebug(t, c, projectName, tu.UniqueString("repo")) }`.  Makes it clearer that we're running variants of the same test.

Reply:
Yeah, although it does make for longer tests, it probably makes the test _output_ more legible, which is a win.